<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>253</width>
    <height>311</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Work with controller</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/1/icons/notification.png</normaloff>:/1/icons/notification.png</iconset>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QVBoxLayout" name="indicatorsLayout"/>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <widget class="QLabel" name="statusMessage">
      <property name="text">
       <string>Система активна</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>253</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Послать команду</string>
    </property>
    <widget class="QMenu" name="menu_3">
     <property name="title">
      <string>Другие (резерв)</string>
     </property>
     <addaction name="sendCommand3"/>
     <addaction name="sendCommand5"/>
    </widget>
    <addaction name="restartComAction"/>
    <addaction name="separator"/>
    <addaction name="restartModemAction"/>
    <addaction name="restartAllAction"/>
    <addaction name="disableCommandsAction"/>
    <addaction name="enableCommandsAction"/>
    <addaction name="menu_3"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Дополнительно</string>
    </property>
    <addaction name="settingsAction"/>
    <addaction name="clearLogAction"/>
    <addaction name="separator"/>
    <addaction name="viewTempLogAction"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <action name="settingsAction">
   <property name="text">
    <string>Настройки</string>
   </property>
  </action>
  <action name="restartComAction">
   <property name="text">
    <string>Перезапустить COM</string>
   </property>
  </action>
  <action name="restartModemAction">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Перезагрузить модем (2)</string>
   </property>
  </action>
  <action name="restartAllAction">
   <property name="text">
    <string>Полный перезапуск комплекса (4)</string>
   </property>
  </action>
  <action name="disableCommandsAction">
   <property name="text">
    <string>Отключить приём команд (6)</string>
   </property>
  </action>
  <action name="enableCommandsAction">
   <property name="text">
    <string>Включить приём команд (7)</string>
   </property>
  </action>
  <action name="clearLogAction">
   <property name="text">
    <string>Очистить лог</string>
   </property>
  </action>
  <action name="sendCommand3">
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="sendCommand5">
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="viewTempLogAction">
   <property name="text">
    <string>Просмотреть лог температуры</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
